packages/fxa-content-server/app/styles/modules/_graphic.scss

```diff
@@ -14,7 +14,7 @@
 }
 
 .graphic-checkbox {
-  @include hidpi-background-image('graphic_checkbox', 150px 130px);
+  @include hidpi-background-image('graphic_checkbox2', 150px 130px);
```

Comment:
@shane-tomlinson 

I did the 3 steps I mentioned above - I exported the icon as an SVG too from photoshop - What is your suggestion - should we add the SVG instead of two png hi and low resolutions?

Reply:
>  should we add the SVG instead of two png hi and low resolutions?

Can you paste the SVG into a comment? I'd like to have a look.

Reply:
I created this pen: https://codepen.io/albionahoti/pen/JVEZwd

Reply:
Oh, great, have you tried it in the CSS to see how it looks?

Reply:
I am having trouble to reproduce and see the change locally - I am creating accounts and verifying the account through the email sent but I am not having the view of the screen above.

Can you give me a guide on how to test it locally - have this display shown?
Otherwise, I can do a commit now and you can review and see if that's good?
@shane-tomlinson

Reply:
If you open the verification link in a different browser the screen showing the green checkbox should appear.

Reply:
@shane-tomlinson I will remove the SVG then - in the meantime, I am checking how I can make the SVG valid.